### PR TITLE
chore: Suppress MD013 for code blocks

### PR DESCRIPTION
**Pull request by vapurrmaid**

### CODE REVIEW

**Review by jawnsy — APPROVED** [review]
niiiice :heart:

File: .markdownlint.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "first-line-heading": false,
   "no-inline-html": false,
-  "ol-prefix": false
+  "ol-prefix": false,
+  "MD013": {
+    "code_blocks": false,
```
Comment (jawnsy) [inline .markdownlint.json:6]: seems to be an extra comma here

Comment (vapurrmaid) [reply .markdownlint.json:6]: It's my inner JavaScript :laughing:

Comment (vapurrmaid) [reply .markdownlint.json]: Fixed in https://github.com/cdr/docs/pull/187/commits/52c52a471e283d278672a6789cfb9e03481ab5a0

Comment (jawnsy) [reply .markdownlint.json]: lol you don't need to tell me that, I'm not a cop :stuck_out_tongue:

I trust you :smile:

Comment (vapurrmaid) [reply .markdownlint.json]: I shall avoid superfluous status update comments in the future, TY @jawnsy , you're always fantastic

Comment (jawnsy) [reply .markdownlint.json]: I mean go for it if you want, just don't want you to feel obligated to update me. IMO, the benefit of code reviews is to learn from each other, not as a rubber stamp or an "approval" really. Healthy teams need an underlying foundation of trust. I trust you to have the best interests of the company at heart.

**Pull request merged**